diagnosis/collectlogsdvm.sh

```diff
@@ -39,9 +39,6 @@ echo "[$(date +%Y%m%d%H%M%S)][INFO][$HOSTNAME] Looking for apt logs" | tee -a $T
 mkdir -p $LOGDIRECTORY/apt/
 try_copy_directory_content /var/log/apt/ $LOGDIRECTORY/apt
 
-echo "[$(date +%Y%m%d%H%M%S)][INFO][$HOSTNAME] Dumping system journal" | tee -a $TRACEFILENAME
```

Comment:
Why is system journal from dvm removed?

Reply:
Reasoning was that it can turn to be a large file and we are not looking anything in particular, we collect it just in case. It may be hard to justify.